### DISCUSSION TITLE
Boost system questions

### DISCUSSION THREAD

**Discussion by cholushkin** [JustInvoke/Randomation-Vehicle-Physics#55] [General]
Hi, 
How to make nitro boost work properly?
Seems like holding B gives only visual effect (blue particles) but engine RPM is not affected (or I don't see difference). 

Why boost increase is calculating like this ?:
float boostEval = boostPowerCurve.Evaluate(Mathf.Abs(vp.localVelocity.z));

**Comment by JustInvoke** [comment]
The way it works is that the boost power scales based on how fast you're driving. I don't remember exactly why I implemented it this way but it must have been because I thought it felt better. You can raise the curve to increase the power, or add a new public float variable to represent the boost power as a fixed value and replace boostEval references with that.
